dask/array/tests/test_array_core.py

```diff
@@ -297,20 +297,17 @@ def test_Array_numpy_gufunc_call__array_ufunc__01():
     x = da.random.normal(size=(3, 10, 10), chunks=(2, 10, 10))
     nx = x.compute()
     ny = np.linalg._umath_linalg.inv(nx)
-    y = np.linalg._umath_linalg.inv(x, output_dtypes=float)
-    vy = y.compute()
```

Comment:
Getting rid of the `compute` is just a clean-up since `assert_eq` will handle that.